src/mon/Paxos.h

```diff
@@ -1065,6 +1065,11 @@ class Paxos {
 		   clock_drift_warned(0),
 		   trimming(false) { }
 
+  ~Paxos() {
+    if (logger)
+      delete logger;
```

Comment:
nit, no need to check for null before `delete`. as `delete logger` would be a no-op if `logger` is nullptr.

Reply:
Ack, thanks.